src/pages/reports/project-report/tabs/ReportData.tsx

```diff
@@ -90,46 +90,46 @@ const ReportDataTab = ({ report, dueAt }: ReportOverviewTabProps) => {
         <PageColumn>
           <PageCard title={Framework.HBF ? t("General Report Updates") : t("Reported Data")} gap={8}>
             <ContextCondition frameworksShow={[Framework.HBF]}>
-              <LongTextField title={t("Landscape Progress")}>{report.landscape_community_contribution}</LongTextField>
-              <LongTextField title={t("Community Engagement Progress")}>{report.community_progress}</LongTextField>
-              <LongTextField title={t("Climate Resilience Progress")}>{report.resilience_progress}</LongTextField>
+              <LongTextField title={t("Landscape Progress")}>{report.landscapeCommunityContribution}</LongTextField>
```

Comment:
There may be other cases in this PR where this is true, so please do a check: make sure that `report` here is correctly typed. I see in this component it's still specified as `any` - one of the big benefits of v3 is getting read of those `any`s and replacing them with things like `ProjectReportFullDto`